packages/next/src/client/components/react-dev-overlay/internal/components/Toast/styles.ts

```diff
@@ -1,29 +1,47 @@
 import { noop as css } from '../../helpers/noop-template'
 
 const styles = css`
-  [data-nextjs-toast] {
+  .nextjs-toast {
     position: fixed;
     bottom: var(--size-gap-double);
     left: var(--size-gap-double);
     max-width: 420px;
     z-index: 9000;
+    box-shadow: 0px var(--size-gap-double) var(--size-gap-quad)
+      rgba(0, 0, 0, 0.25);
   }
 
   @media (max-width: 440px) {
-    [data-nextjs-toast] {
+    .nextjs-toast {
       max-width: 90vw;
       left: 5vw;
     }
   }
 
-  [data-nextjs-toast-wrapper] {
+  .nextjs-toast-errors-parent {
     padding: 16px;
-    border-radius: var(--size-gap-half);
+    border-radius: var(--size-gap-quad);
     font-weight: 500;
     color: var(--color-ansi-bright-white);
     background-color: var(--color-ansi-red);
-    box-shadow: 0px var(--size-gap-double) var(--size-gap-quad)
-      rgba(0, 0, 0, 0.25);
+  }
+
+  .nextjs-static-indicator-toast-wrapper {
+    padding: 8px 16px;
+    border-radius: var(--size-gap-triple);
+    background: var(--color-background);
+    color: var(--color-font);
+  }
+
+  .nextjs-static-indicator-toast-wrapper div {
```

Comment:
From the video, it seems the text is not centered in the indicator, because we're possibly missing a `justify-content: center` here?

Reply:
Our figma design doesn't have it centered to give space from the `X` and `Learn more`

Reply:
I mean the text aligned vertically (space top and below), not affecting `X` or `Learn more`

Reply:
Hmm everything looks vertically aligned properly 🤔 

![CleanShot 2024-07-31 at 15 16 38@2x](https://github.com/user-attachments/assets/ae3cd7ac-8226-4f72-ad6f-74f7f600b06e)

Reply:
Interesting <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture>